windows/src/engine/keyman32/appint/aiWin2000Unicode.h

```diff
@@ -1,18 +1,18 @@
 /*
```

Comment:
Please rollback whitespace-only changes in this file.

Reply:
done